crossing-training.lic

```diff
@@ -684,7 +689,8 @@ class CrossingTraining
     if DRSkill.getxp('Athletics') < 29
       walk_to(2245)
       if UserVars.athletics < 550
-        practice_branch
+        @climbing_target['target'] = 'branch'
```

Comment:
Does this work if you don't have a climbing_target set? I expected this line to throw an error since `@climbing_target` would be `nil`.